charts/matrix-recorder/values.yaml

```diff
@@ -21,4 +21,4 @@ service:
   port: 80
 
 certificate:
-  enabled: true
+  enabled: false
```

Comment:
Why are you removing this? We want to access through https://archive.web3.foundation